apps/src/templates/studioHomepages/TwoColumnActionBlock.jsx

```diff
@@ -21,14 +21,22 @@ const styles = {
     minHeight: 260,
     boxSizing: 'border-box'
   },
-  subheading: {
+  subHeading: {
     paddingRight: 0,
     paddingBottom: 20,
     fontSize: 27,
     lineHeight: 1.2,
     fontFamily: '"Gotham 7r", sans-serif',
     color: color.white
   },
+  subHeadingSmallFont: {
+    paddingRight: 0,
+    paddingBottom: 20,
+    fontSize: 25,
+    lineHeight: 1.2,
+    fontFamily: '"Gotham 7r", sans-serif',
+    color: color.white
+  },
```

Comment:
We could extract the common styles between `subHeading` and `subHeadingSmallFont` and then set the font size only with the conditional below, but I don't feel strongly about it.